packages/driver/src/cypress/chai_jquery.ts

```diff
@@ -116,10 +120,11 @@ export const $chaiJquery = (chai, chaiUtils, callbacks: Callbacks) => {
       a = a.not
     }
 
+    // @ts-expect-error - TODO: Fix this
```

Comment:
I think if we type args as `args: [string, string?]` this will allow this to work properly. Not sure if that would break people's types though?